src/main/java/hudson/plugins/ec2/util/MinimumInstanceChecker.java

```diff
@@ -119,7 +119,7 @@ public static void checkForMinimumInstances() {
 
     public static boolean minimumInstancesActive(
         MinimumNumberOfInstancesTimeRangeConfig minimumNumberOfInstancesTimeRangeConfig) {
-        if (minimumNumberOfInstancesTimeRangeConfig == null) {
+        if (minimumNumberOfInstancesTimeRangeConfig == null || minimumNumberOfInstancesTimeRangeConfig.getMinimumNoInstancesActiveTimeRangeDays() == null) {
```

Comment:
For the function `getMinimumNoInstancesActiveTimeRangeDays()` you might want to annotate it with `@CheckForNull`